data/anope.example.conf

```diff
@@ -134,22 +134,31 @@ define
  *     <uline server="services.localhost.net" silent="yes">
  *     <bind address="127.0.0.1" port="7000" type="servers">
  *
- * An example configuration for UnrealIRCd that is compatible with the below uplink
+ * An example configuration for UnrealIRCd 4.x that is compatible with the below uplink
  * and serverinfo configuration would look like:
  *
- *     link services.localhost.net
- *     {
- *          username *;
- *          hostname *;
- *          bind-ip "127.0.0.1";
- *          port 7000;
- *          hub *;
- *          password-connect "mypassword";
- *          password-receive "mypassword";
- *          class servers;
- *     };
- *     ulines { services.localhost.net; };
- *     listen 127.0.0.1:7000;
+ * listen {
+ *   ip 127.0.0.1;
+ *   port 7000;
+ *    options {
+ *     serversonly;
+ *    };
+ * };
+ *
+ * link services.localhost.net
+ * {
```

Comment:
This should probably be on the same line as "link" for consistency

Reply:
The listen block is fine. I have it like that and as per UnrealIRCd docs this is how it should be.

Reply:
Oh sorry, I misunderstood the point here.
Perhaps the `{` should be on the same line, yes. but I just copied the example from UnrealIRCd docs.